File: C/06-Operadores Relacionais/Relacionais.c
#include<stdio.h>

int main(void)
{
    //Operadores relacionais testam uma relação entre variaveis(se são verdadeiras ou falsas)
    //Se for verdadeira retorna 1 se for falsa retorna 0.
    int s = 2;
    int t = 3;
    int v = 2;

    //Os operadores relacionais em c são >(maior) >=(maior ou igual) <(menor) <=(menor ou igual)
    //==(igual) !=(diferente)

    printf("1 = Verdadeiro / 0 = Falso \n");
    printf("%d > (maior) %d ? %d \n", s, t, s > t);//falso
    printf("%d < (menor) %d ? %d \n", s, t, s < t);//verdadeiro
    printf("%d >= (maior ou igual) %d ? %d \n", s, v, s >= v);//verdadeiro
    printf("%d <= (menor ou igual) %d ? %d \n", t, s, t <= s);//falso
    printf("%d == (igual) %d ? %d \n", s, v, s == v);//verdadeiro
    printf("%d != (diferente) %d ? %d \n", s, t, s != t);//verdadeiro
    return 0;
}

File: PHP/04-Operadores Relacionais/Relacionais.php
<?php

//Os operadores relacionais testam a relação entre variaveis
//Os operadores relacionais em php são
// < menor
// > maior
// <= menor ou igual
// >= maior ou igual
// == igual
// === identico(tipo também deve ser igual)

$x = 2;
$y = 4;
$z = '2'; //String

echo"1 - VERDADEIRO / [NDA] - FALSO";
echo'<br/>';

echo $x . " <(menor que) " . $y . " : ";
echo'<br/>';
echo $x < $y;
echo'<br/>';

echo $x . " >(maior que) " . $y . " : ";
echo'<br/>';
echo $x > $y;
echo'<br/>';

echo $x . " <=(menor ou igual que) " . $y . " : ";
echo'<br/>';
echo $x <= $y;
echo'<br/>';

echo $x . " >=(maior ou igual que) " . $z . " : ";
echo'<br/>';
echo $x >= $z;
echo'<br/>';

echo $x . " ==(igual que) " . $y . " : ";
echo'<br/>';
echo $x == $y;
echo'<br/>';

echo $x . " ===(identico que) ''" . $z . "'' : ";
echo'<br/>';
echo $x === $z; //São de tipos diferentes
echo'<br/>';

?>